pkg/config/config.go

```diff
@@ -408,7 +406,10 @@ func (f *File) expandFiles(dir string) ([]*File, error) {
 		}
 
 		fileNames, err := filepath.Glob(*f.Path)
-		if err != nil || len(fileNames) == 0 {
+		if len(fileNames) == 0 && err == nil {
+			err = fmt.Errorf("file: %s doesn't exist", *f.Path)
+		}
+		if err != nil {
```

Comment:
How about?
```golang
      if err != nil {
				return err
			}
			if len(fileNames) == 0 {
				return &os.PathError{Op: "open", Path: *f.Path, Err: fs.ErrNotExist}
			}
```

Reply:
it will be much better, I will resolve it promptly.

Reply:
@veezhang I have improved it， please check it again.

Reply:
@LooJee Good job!
Sorry, I find the ci is use go1.13, so maybe you should use `os.ErrNotExist` instead of `fs.ErrNotExist`.